Conductor/conf/supervisord.conf

```diff
@@ -36,7 +36,7 @@ startsecs=0
 exitcodes=0
 
 [program:nginx]
-command=bash -c "sleep 10 && /usr/sbin/nginx -g 'daemon off;'"
```

Comment:
If we use variables for upstreams, Nginx doesn't crash but returns a 502 (Bad Gateway) response until the upstream is available.